src/default_out.rs

```diff
@@ -85,24 +85,16 @@ pub fn dump_undefined_struct(
                 }
             }
 
-            if let Some(rom_size) = data.rom_size() {
-                if rom_size != 0xFF {
-                    println!("\tROM Size: {} {}", ((rom_size + 1) as u64) << 6, KB);
-                } else {
-                    if let Some(extended_rom_size) = data.extended_rom_size() {
-                        print!("\tROM Size: ");
-                        match extended_rom_size {
-                            ExtendedRomSize::Megabytes(size) => {
-                                println!("{} {}", size, MB);
-                            }
-                            ExtendedRomSize::Gigabytes(size) => {
-                                println!("{} {}", size, GB);
-                            }
-                            ExtendedRomSize::Undefined(size) => {
-                                println!("{} ({})", OUT_OF_SPEC, size);
-                            }
-                        }
-                    }
+            let rom_size = data.extended_rom_size().or(data.rom_size());
```

Comment:
I like the idea of using the .or() on these extended cases.  We can borrow this pattern in the dmi_cache_size() code refactoring I did and reduce some of the match arms.